core/procedural-primitives/procedural-primitives.zig

```diff
@@ -185,12 +185,16 @@ pub fn createCubePrimitive(allocator: std.mem.Allocator, size: f32) !Primitive {
 }
 
 pub fn createCylinderPrimitive(allocator: std.mem.Allocator, radius: f32, height: f32, num_sides: u32) !Primitive {
```

Comment:
Hi @BlackGoku36, if I understand this correctly num_sides means how many vertices will top and bottom circles of a cylinder have, right?

We could probably name it as `vertices: u32` as in a circle, or that we have some unified naming. I took this name from the blender.

Here is an example from blender so that you will know what I am talking about
![blender-procedural-cylinder](https://user-images.githubusercontent.com/8342378/226839428-922abea7-7194-4a5f-a0c3-6c6350b29560.gif)

Reply:
I think vertices would be bit confusing for users as they might think vertices as in 'total vertices', but sides is more clearer. Like when we talk about polygon, we usually talk about polygon with n-sides (4=quadrilateral, 5=pentagon, 6=hexagon, etc). That why I named it sides. I don't know why blender did that, it did confuse me a little at first. But if you want to still change, than I don't really have problem with it.

Reply:
Aaaa I see your point, could we do then polygon-sides or something in this direction?
Or maybe n-edges?

Because when I saw sides I thought that you mean top and bottom faces.